### PR TITLE
Added self paced pl dropdown list

### PR DESCRIPTION
**Pull request by juanmanzojr**
Adds the Self-Paced Professional Learning Course Offering Dropdown List which saves the corresponding `self_paced_pl_course_offering_id`. Update the corresponding test file to include `self_paced_pl_course_offering_id`. Also removes the `course_version_path` of the self paced professional learning course offerings from the summary as there is no need for them anymore.

### Saves in db
Screenshot 2023-07-26 at 10 18 59 AM

Screenshot 2023-07-26 at 10 19 09 AM

## Links

- jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?modal=detail&selectedIssue=ACQ-653)

## Testing story
Local and Unit test

## Follow-up work
I will work on adding the last two fields, `video` and `date_published`.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/levelbuilder/CourseOfferingEditor.jsx [hunk 1]
```diff
@@ -345,29 +345,61 @@ export default function CourseOfferingEditor(props) {
...
+          <p>
+            Pick the self-paced course offering that supports teachers teaching
+            the course offering.
+          </p>
+        </HelpTip>
+        <select
+          value={courseOffering.self_paced_pl_course_offering_id}
+          style={styles.dropdown}
+          onChange={e => {
+            console.log(e.target.value);
```
Comment (TurnerRiley) [inline apps/src/lib/levelbuilder/CourseOfferingEditor.jsx]: Nit: remove the console.log call please

**Review by TurnerRiley — CHANGES_REQUESTED** [review]
I'm not sure if this just requires merging in the other branch where you allow the nil entry when the "(None)" option is selected, but I'm getting the `Error Saving: Validation failed: Professional learning program must be one of the professional learning program path. Expected one of: ["code.org/professional-development-workshops", "code.org/apply"]. Got: "".` when I try to switch the selection back to the "(None)" option

**Review by TurnerRiley — COMMENTED** [review]
Sorry about the self-request of the review and the red blocker, I clicked "Request changes" when I should've clicked "Comment" before

File: apps/src/lib/levelbuilder/CourseOfferingEditor.jsx (hunk 1 above)
Comment (juanmanzojr) [reply apps/src/lib/levelbuilder/CourseOfferingEditor.jsx]: Sorry about this ill make these changes, and yes I just merged the nil entry acceptance PR into staging so I will merge into this PR

**Review by TurnerRiley — APPROVED** [review]
Sweet, looks great!

**Pull request merged**